glue/src/it/scala/com/gu/tableversions/glue/GlueMetastoreSpec.scala

```diff
@@ -0,0 +1,77 @@
+package com.gu.tableversions.glue
+
+import java.net.URI
+
+import cats.effect.IO
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.auth.{
+  AWSCredentialsProviderChain,
+  EnvironmentVariableCredentialsProvider,
+  InstanceProfileCredentialsProvider,
+  SystemPropertiesCredentialsProvider
+}
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.glue.model._
+import com.amazonaws.services.glue.{AWSGlue, AWSGlueClient}
+import com.gu.tableversions.core.Partition.PartitionColumn
+import com.gu.tableversions.core._
+import com.gu.tableversions.metastore.MetastoreSpec
+import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
+import cats.implicits._
+
+class GlueMetastoreSpec extends FlatSpec with Matchers with BeforeAndAfterAll with MetastoreSpec {
+
+  lazy val region = Regions.EU_WEST_1
+
+  lazy val credentials = new AWSCredentialsProviderChain(
+    new EnvironmentVariableCredentialsProvider(),
+    new SystemPropertiesCredentialsProvider(),
+    new ProfileCredentialsProvider("ophan"),
```

Comment:
I'm not sure about this, as it is Guardian specific... is your current plan to run this test from TeamCity?

Reply:
I'll just write a recap here of our talk on friday:
When writing this I assumed Teamcity would not be able to run integration tests but if it can be given access to an aws account it would be better. Otherwise we could just run it locally and maybe add a trigger so that it runs before pushing or something to like that to avoid people forgetting to run it.

Reply:
Regarding referencing the ophan profile explicitly, we could extract the profile name to a configuration file or env variable or something